.github/actions/test-khiops-on-iris/action.yml

```diff
@@ -45,6 +48,7 @@ runs:
         KHIOPS_PROC_NUMBER: 4
```

Comment:
If we set this variable, then we don't test that the new binary _getprocnumber works, do we?